src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm \n" + logo );
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        Scanner scanner = new Scanner(System.in);
+        String text = "";
+        Boolean isContinue = true;
+
+        Todo[] todos = new Todo[100];
+        Integer index = 0;
+
+        while (isContinue) {
+            text = scanner.nextLine();
+            if (text.equals("list")) {
+                System.out.println("____________________________________________________________");
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < index; i++){
+                    int number = i+1;
+                    System.out.println( number + "." + todos[i]);
+                }
+                System.out.println("____________________________________________________________");
+
+            } else {
+                String[] arr = text.split(" ", 2);
+                String action = arr[0];
+                String input = arr[1];
+
+                if (action.equals("done")){
+                    int setAsDone =Integer.parseInt(input) ;
```

Comment:
I think the int variable here is for the index of the task that is about to be set as done. Using "setAsDone" as the name of the int variable is a bit misleading.